docs/focal_point_plasticity.rst

```diff
@@ -122,6 +122,34 @@ example):
    </Plugin>
 
 
+The total number of links a given cell can form is computed as a sum of ``<MaxNumberOfJunctions>`` values coming from ``<Parameters>``
+or ``<InternalParameters>`` sections where a given cell type appears. In the example above ``Center`` can form maximum of 2 internal links (between compartments)
+
+CompuCell3D gives you ability to override this default algorithm by adding additional tag that control the total number of links:
```

Comment:
nitpick: This could say, "the total number of links of a given cell type." It took me a moment to realize that it's not possible to control the global, simulation-wide number of links. Instead, "total" just pertains to each cell.